README.md

```diff
@@ -1,49 +1,59 @@
 # Prerequisites
 
-- [Docker](https://docs.docker.com/docker-for-mac/install/)  
+- [Docker](<https://docs.docker.com/docker-for-mac/install/>)
```

Comment:
What markdown format is this? I don't think the angle brackets are common https://guides.github.com/features/mastering-markdown/#examples

Reply:
yeah, stumbled upon this as well.